ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckWithCommittedImpl.scala

```diff
@@ -127,7 +127,6 @@ private[validate] class ConflictCheckWithCommittedImpl(
         f.flatMap {
           case Right(_) =>
             indexService
-              // TODO SoX: Perform lookup more efficiently and do not use a readers-based lookup
```

Comment:
In the happy flow, this should be a cache hit hence no further optimization needed.